develop/dev-guide-choose-driver-or-orm.md

```diff
@@ -143,7 +147,48 @@ In addition, you need to specify the TiDB dialect in your [Hibernate configurati
 >
 > If you are unable to upgrade your `Hibernate` version, use the MySQL 5.7 dialect `org.hibernate.dialect.MySQL57Dialect` instead. However, this setting might cause unpredictable results and the absence of some TiDB-specific features, such as [sequences](/sql-statements/sql-statement-create-sequence.md).
 
-### tidb-loadbalance
+</div>
+
+<div label="MyBatis">
+
+Support level: **Full**
+
+To avoid manually managing complex relationships between different dependencies of an application, you can use [Gradle](https://gradle.org/install) or [Maven](https://maven.apache.org/install.html) to get all dependencies of your application, including those indirect ones.
```

Comment:
```suggestion
To avoid manually managing complex relationships between different dependencies of an application, you can use [Gradle](https://gradle.org/install) or [Maven](https://maven.apache.org/install.html) to get all dependencies of your application, including those indirect dependencies.
```